src/PowerModelsGMD.jl

```diff
@@ -1,15 +1,17 @@
 module PowerModelsGMD
+    
+    import PowerModels
+    import Ipopt
     import JuMP
 
     import InfrastructureModels
-    import PowerModels
-
     import Memento
 
     using Printf: @printf, @sprintf
 
     const LOGGER = Memento.getlogger(PowerModels)
     const PMs = PowerModels
+    const PMgmd = PowerModelsGMD
```

Comment:
Normally, there is no need to define a constant like this becouse all of the things defined within a module are in the scope of that module and do not need to be qualified with a prefix.

Reply:
@ccoffrin, you are absolutely correct! However, for the clarity of the (under development) code, decided to add this simplifying constant and reference certain variables/constraints/etc. to know where they are coming from.

Reply:
Ok, you make a good point.  The new standard in Julia packages is that they should use `import` of other packages.  This way all external functions must be qualified with `<package>.<function>`, then anything without a qualifier is internal.